src/Sql/Sql.Test/ScenarioTests/ThreatDetectionTests.ps1

```diff
@@ -272,6 +290,10 @@ function Test-InvalidArgumentsThreatDetection
 		#  Check that NotificationRecipientsEmails are in correct format 
 		Assert-Throws {Update-AzSqlDatabaseAdvancedThreatProtectionSetting -ResourceGroupName $params.rgname -ServerName $params.serverName -DatabaseName $params.databaseName -NotificationRecipientsEmails "kokogmail.com"} 
 
+		#  Check that EmailAdmins is not False and NotificationRecipientsEmails is not empty 
+		Assert-Throws {Update-AzSqlDatabaseAdvancedThreatProtectionSetting -ResourceGroupName $params.rgname -ServerName $params.serverName -DatabaseName $params.databaseName -EmailAdmins $false -StorageAccountName $params.storageAccount -StoragEndPoint $params.storageEndpoint} 
+		Assert-Throws {Update-AzSqlDatabaseAdvancedThreatProtectionSetting -ResourceGroupName $params.rgname -ServerName $params.serverName -DatabaseName $params.databaseName -EmailAdmins $false -NotificationRecipientsEmails "" -StorageAccountName $params.storageAccount -StoragEndPoint $params.storageEndpoint} 
```

Comment:
not related to your change

Reply:
It basically tests that the test ran properly.. Maybe it's not needed but I'll keep it in any case for later use - if something breaks in the creation of the DB settings it will fail here